src/pages/resultsView/mutualExclusivity/MutualExclusivityTable.tsx

```diff
@@ -132,17 +132,17 @@ export default class MutualExclusivityTable extends React.Component<IMutualExclu
             download: (d: MutualExclusivity) => formatPValue(d.pValue)
         };
 
-        this._columns[MutualExclusivityTableColumnType.ADJUSTED_P_VALUE] = {
-            name: "Adjusted p-Value",
-            render: (d: MutualExclusivity) => formatPValueWithStyle(d.adjustedPValue),
-            tooltip: <span>Bonferroni adjusted p-Value</span>,
-            sortBy: (d: MutualExclusivity) => d.adjustedPValue,
-            download: (d: MutualExclusivity) => formatPValue(d.adjustedPValue)
+        this._columns[MutualExclusivityTableColumnType.Q_VALUE] = {
+            name: "q-Value",
+            render: (d: MutualExclusivity) => formatQValueWithStyle(d.qValue),
+            tooltip: <span>Derived from Benjamini-Hochberg procedure</span>,
```

Comment:
might be good to say "Benjamini-Hochberg FDR correction procedure", because not everyone will know by looking immediately